### ISSUE TITLE
Accessibility Inspections For HTML

### ISSUE THREAD

**Issue by auchenberg** [microsoft/vscode-html-languageservice#54]
Customer request from https://dev.to/mokkapps/why-i-switched-from-visual-studio-code-to-jetbrains-webstorm-939

WebStorm provides inspections which are based on recommendations from Web Content Accessibility Guidelines (WCAG) which help you to write more accessible HTML code.

![image](https://user-images.githubusercontent.com/173559/53279220-6440c600-36c3-11e9-808a-6bc07ad02f95.png)

**Comment by coliff** [comment]
sounds like a good job for an extension. [htmlhint ](https://marketplace.visualstudio.com/items?itemName=mkaufman.HTMLHint)does this already.

**Comment by aeschli** [comment]
Closing, we have no plans for adding validation. Should be left to linters.
